bin/README.md

```diff
@@ -1,6 +1,6 @@
 # Command Line Tools
 
-VPX-JS comes with two command line tools:
+VPX-JS comes with command line tools. Make sure to run `npm run compile` first, then change to the compiled output directory (`dist/cjs/bin`) when you use Node.js. Another option is transpile the TypeScript file on the fly and execute it using `npx ts-node bin/vptscript.ts`.

```

Comment:
I don't think you need to do that if you get vpx-js from npm?

Reply:
i guess if you take the package from npm you don't read the readme. 

if we have some tools bundled with vpx-js, then it should be clear how run them - else it is not very helpful.